### PR TITLE
Add support for Apache Mesos operator

### PR DESCRIPTION
**Pull request by pgillet**
Hi douban,

I added support for Apache Mesos operator API.
See [Operator HTTP API Document](http://mesos.apache.org/documentation/latest/operator-http-api/)

Would you consider merging this PR?
I'm waiting for your feedback.

### CODE REVIEW

File: pymesos/operator_v1.py [hunk 1]
```diff
@@ -0,0 +1,558 @@
...
+    def getTasks(self):
+        body = dict(
+            type='GET_TASKS',
+        )
+        return self._send(body)
+
+
+class MesosOperatorMasterDriver(Process, MesosOperatorDaemonDriver):
+    """
+        For now, supports only operations that retrieve data.
```
Comment (windreamer) [inline pymesos/operator_v1.py]: Why? I found methods such like `createVolumes`, `setQuota` implemented. So we can update status of master as well as just retrieve data?

File: pymesos/subprocess/operator_v1.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+from .. import OperatorMaster
```
Comment (windreamer) [inline pymesos/subprocess/operator_v1.py]: Why we need this?

File: tests/test_operator.py [hunk 3]
```diff
@@ -0,0 +1,1179 @@
...
+                'heartbeat': {
+                    'interval': {
+                        'nanoseconds': 30000000000
+                    }
+                }
+            }
+        }
+    ]
+    driver.attachContainerInput(container_id, process_ios)
+    awaited_result = '153\n{"type": "ATTACH_CONTAINER_INPUT", "attach_container_input": {"type": "CONTAINER_ID", ' + \
```
Comment (windreamer) [inline tests/test_operator.py]: Tests failed here: the order of `type` field and `container_id` field is indeterministic.
We may do some explicit checking here?

File: pymesos/operator_v1.py (hunk 1 above)
Comment (pgillet) [reply pymesos/operator_v1.py]: It was an old forgotten comment. I deleted it. The Operator API is now 100% covered.

File: pymesos/subprocess/operator_v1.py (hunk 2 above)
Comment (pgillet) [reply pymesos/subprocess/operator_v1.py]: We don't actually: the ProcOperatorMaster class is not used anywhere.
The passed OperatorMaster in the MesosOperatorMasterDriver constructor is optional: if no OperatorMaster is passed, then the event methods are not intended to be called.

But we may think of a different behavior: if no OperatorMaster is passed, then the default ProcOperatorMaster is used which just logs the Mesos callbacks.

Make your choice, and I'll do the necessary changes if any.

Comment (windreamer) [reply pymesos/subprocess/operator_v1.py]: Hmm... I think we do not need this for now.  `pymesos.subprocess` is an in-place replacement for `subprocess` on Mesos, thus we do not need an `Operator` implementation.

File: tests/test_operator.py (hunk 3 above)
Comment (pgillet) [reply tests/test_operator.py]: I corrected the test case to get the same ordering in the output json.

**Comment by pgillet** [comment]
On which version of Python should I install Flake8?

**Comment by pgillet** [comment]
I reformatted the code following Flake8

**Comment by pgillet** [comment]
:+1:

**Pull request merged**